src/pages/blog/sigcse2024_AffiliatedEvent/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import classnames from "classnames";
+import Head from "next/head";
+
+import { PageBanner } from "../../../components/Banner";
+import { Heading } from "../../../components/Heading";
+import Stack from "../../../components/Stack";
+
+export default function SIGCSE2024() {
+  return (
+    <React.Fragment>
+      <Head>
+        <title>SIGCSE 2024</title>
+      </Head>
+
+      <PageBanner
+        title="SIGCSE 2024"
+        subtitle="Technical Symposium on Computer Science Education"
+      />
+
+      <div className={classnames("container-fluid py-4")}>
+        <div className="container-md pt-2">
+          <Stack spacing={4}>
+            <Heading>
+              Affiliated Event: Spiffy Assessments with PrairieLearn
+            </Heading>
+
+            <p>
+              A key element in both mastery learning and competency-based
+              education involves providing students with multiple opportunities
+              to practice with rapid feedback, a feature provided by
+              autograders. PrairieLearn is an open-source assessment authoring
+              system that empowers the development of exceptionally rich
+              interactive STAR questions &#8212; Skill-specific, Tagged to
+              course competencies, Auto-graded, and Randomizable.
+            </p>
+            <p>
+              Questions are formulated as problem generators capable of
+              producing multiple variants of a question. This allows question
+              authors to utilize the same question generator across different
+              assignments and semesters, reducing the workload in the long run
+              and facilitating the implementation of second-chance exams. In
+              addition, this framework empowers students to generate multiple
+              instances of a question, providing them with enhanced practice
+              opportunities.
+            </p>
+            <p>
+              {" "}
+              Currently, PrairieLearn is actively adopted in over 800 courses
+              across 20 universities. In this affiliated event, instructors from
+              these courses will showcase the innovative online assessments
+              they’ve created using PrairieLearn, and they are eager to share
+              these resources with the broader instructor community. Presenters
+              will also be happy to discuss their experience authoring these
+              assessments.
+            </p>
+
+            <h4>Organizers</h4>
+            <ul>
+              <li>
+                Prof. Mariana Silva, University of Illinois Urbana-Champaign.,
+                mfsilva@illinois.edu, https://mfsilva.web.illinois.edu/
```

Comment:
I would encourage you to make the emails + sites actually links (via `<a ...>`).